lib/session.js

```diff
@@ -143,12 +151,34 @@ module.exports = class Session {
     return cookieSignature.sign(this.sessionId, this[secretKey])
   }
 
+  [hash] () {
+    const sess = this
+    const str = stringify(sess, function (key, val) {
```

Comment:
should we store the serialized session (`str` here) as well and use that in `save` instead of relying on whatever does the current serialization in the individual stores?

if yes, separate PR, but still

Reply:
I wouldn't